core/src/test/java/org/apache/iceberg/puffin/TestPuffinFormat.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.puffin;
+
+import java.io.ByteArrayOutputStream;
+import org.junit.Test;
+
+import static org.apache.iceberg.puffin.PuffinFormat.readIntegerLittleEndian;
+import static org.apache.iceberg.puffin.PuffinFormat.writeIntegerLittleEndian;
+import static org.apache.iceberg.relocated.com.google.common.base.Preconditions.checkArgument;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestPuffinFormat {
+  @Test
+  public void testWriteIntegerLittleEndian() throws Exception {
+    testWriteIntegerLittleEndian(0, bytes(0, 0, 0, 0));
+    testWriteIntegerLittleEndian(42, bytes(42, 0, 0, 0));
+    testWriteIntegerLittleEndian(Integer.MAX_VALUE - 5, bytes(0xFa, 0xFF, 0xFF, 0x7F));
```

Comment:
Nit: lower-case `a` should be upper case.

Reply:
right!